documentation/wordcab_api_reference.yaml

```diff
@@ -548,13 +705,136 @@ paths:
+                "uuid": "abc",
+                "user_id": "123"
+            }
+            ```
+          schema:
+            type: string
+          example: X-Wordcab-Metadata-UUID
+        - name: source
+          in: query
+          required: true
+          description: |
+            Declare the source of the data that you want to summarize. 
+            _📝 Note that some sources have different parameter requirements._
+            * `generic` - Generic-format .txt and JSON transcripts
+            * `audio` - For uploaded audio files
+            * `wordcab_transcript` - Summarize Wordcab transcripts using a `transcript_id`
+            * `signed_url` - _AWS_ or _GCP_ signed URLs
+            * `assembly_ai` - Use the JSON array from AssemblyAI's `utterances` object as input
+            * `deepgram` - Use the JSON array from Deepgram's `utterances` object as input
+            * `rev_ai` - Use the JSON array from Rev.ai's `monologues` object as input
+            * `vtt` - Either a .vtt file, or raw vtt text in the request body - Learn more about [proper vtt formatting](https://quuz.org/webvtt/). 
+            _Note that speaker labels are required._
+            * `otter` - Learn more about [Otter.ai transcript formatting](https://wordcab.com/blog/summarizing-your-otterai-transcripts-in-wordcab/)
+            * `fireflies` - Learn more about [Fireflies.ai transcript formatting](https://wordcab.com/blog/summarizing-your-firefliesai-transcripts-in-wordcab/)
+            * `sonix` - Learn more about [Sonix.ai transcript formatting](https://wordcab.com/blog/summarizing-your-sonixai-transcripts-in-wordcab/)
+            * `descript` - Learn more about [Descript transcript formatting](https://wordcab.com/blog/how-to-summarize-descript-transcripts-in-wordcab/)
+          schema:
+            type: string
+            default: generic
+            enum: [generic, audio, wordcab_transcript, signed_url, assembly_ai, deepgram, rev_ai, vtt, otter, fireflies, sonix, descript]
+          example: generic
+        - name: display_name
+          in: query
+          required: false
+          description: |
+            A human-readable name for your job. Not strictly required, but highly recommended.
+          schema:
+            type: string
+          example: my_extract_job
+        - name: pipeline
+          in: query
+          required: false
+          description: >
+            **Default value:** `questions_answers,topic_segments,emotions,speaker_talk_ratios`<br>
+            A comma-separated list of actions to take upon initiating the job. You can specify any combination of the following:
+            * `questions_answers` - Detect question-answer pairs throughout the transcript.
+            * `topic_segments` - Extract topic segments from the transcript
+            * `emotions` - Extract emotions from the transcript
+            * `speaker_talk_ratios` - Details of the amount of time each speaker spoke
```

Comment:
Is there any default value, or is this parameter required?

Reply:
Default value is in the description, it would all of them methods

Reply:
Yes, I added this default value in the description, but I was wondering if it was all the pipelines by default. :)

This review can be closed, and the PR can be merged then. The extract point is ready!